src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -173,9 +204,14 @@ def on_time_step(self, event: Event):
             address_id_col_name="employer_address_id",
         )
 
-        # change jobs if of working age already and either selected by rate or moved
-        # on this timestep
-        working_age_idx = pop.loc[pop["age"] >= data_values.WORKING_AGE].index
+        # change jobs if of working age already but exclude simulants living in military group quarters
+        working_age_idx = pop.index[
```

Comment:
Is this change related to the other changes in this PR? If so, I don't immediately see how.

Reply:
Yes.  This is excluding military simulants otherwise they were potentially changing jobs >> getting a new employer propensity >> then being assigned back to military employer.  That is why this was changed and military index chunk of code was moved.

Reply:
Ah, makes sense! Carry on.